src/lib.rs

```diff
@@ -282,7 +301,7 @@ struct Post<BlockNumber, Moment, AccountId> {
     thread_id: ThreadId,
 
     // Position of post in thread.
-    position: u64,
+    position: u32,
```

Comment:
I have already responded to this in the prior review for https://github.com/Joystream/substrate-forum-module/pull/1, please review and respond to that if you disagree. Again this was also in the spec, and should have been raised then.

Reply:
thanx for reminding about your previous reply to it.
Yes, I still see problems with `position`, because:
- thread/post can be moderated
- post can be pinned

All these will make a direct impact on the order of threads/posts on any certain page N.

Reply:
As discussed, it is needed, but the name is confusing, as you interpreted it as a UI concept, which it was not. I have the position field for both `Post` and `Thread`.